file-formats/aifr/type/zif_aff_aifr_v1.intf.abap

```diff
@@ -0,0 +1,27 @@
+INTERFACE zif_aff_aifr_v1
+  PUBLIC.
+  TYPES:
+    "! <p class="shorttext">Recipient Details</p>
+    "! Recipient details
+    BEGIN OF ty_recipient,
+      "! <p class="shorttext">Namespace Object Name</p>
+      "! Namespace object name of the recipient
+      "! $required
+      namespace_object_name TYPE c LENGTH 15,
+    END OF ty_recipient,
+
+    "! <p class="shorttext">Recipient</p>
+    "! Recipient
```

Comment:
```suggestion
    "! Application Interface Framework Recipient
```

Reply:
Since we want to keep the name and description of the repository object/development object in sync with the object in the ADT editor, we will stay with the name Recipient.